kubernetes/samples/scripts/create-weblogic-domain/model-in-image/sample-model/JRF/model.10.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+domainInfo:
+    AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
+    AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
+    ServerStartMode: 'prod'
+    RCUDbInfo:
+        rcu_prefix: '@@SECRET:@@ENV:DOMAIN_UID@@-rcu-access:rcu_prefix@@'
+        rcu_schema_password: '@@SECRET:@@ENV:DOMAIN_UID@@-rcu-access:rcu_schema_password@@'
+        rcu_db_conn_string: '@@SECRET:@@ENV:DOMAIN_UID@@-rcu-access:rcu_db_conn_string@@'
+
+topology:
+    AdminServerName: 'admin-server'
+    Name: '@@ENV:CUSTOM_DOMAIN_NAME@@'
+    Cluster:
+        'cluster-1':
+    Server:
+        'admin-server':
+            ListenPort: 7001
+        'managed-server1':
+            Cluster: 'cluster-1'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+         'managed-server2':
+            Cluster: 'cluster-1'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+         'managed-server3':
+            Cluster: 'cluster-1'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+         'managed-server4':
+            Cluster: 'cluster-1'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+           
+appDeployments:
+    Application:
+        myear:
+            SourcePath: 'wlsdeploy/applications/myapp'
```

Comment:
Are we packaging as exploded format or ear format?

Reply:
Exploded.

Reply:
Please resolve if your fine with this.